perfscale_regression_ci/scripts/scalability/replace_loadedworker_nodes.sh

```diff
@@ -0,0 +1,69 @@
+#/!/bin/bash
+##################################################################################################
+## Auth=svetsa@redhat.com
+## Desription: Script for resizing worker nodes with larger worker nodes
+## Polarion test case: OCP-40557 - Replace loaded worker nodes with larger instances
+## https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-40557
+## Customer related: https://access.redhat.com/support/cases/#/case/02864921
+## Pre-requisites: 
+### Cluster with 3 m5.2xlarge master nodes and 30 m5.xlarge worker cluster on AWS, or equivalent size of these profiles:
+#### AWS: IPI-on-AWS-OVN
+#### GCP: IPI-on-GCP-OVN
+#### Alicloud: IPI-on-AlibabacloudOVN
+## kube-burner config: 
+### e2e-benchmarking/workloads/kube-burner/workloads/cluster-density/cluster-density.yml
+## Parameters: Cloud provider, number of replicas and number of JOB_ITERATIONS
+###################################################################################################
+
+source ./replace_loadedworker_nodes_env.sh
+source ../common.sh
+source ../../utils/run_workload.sh
+
+# If parameters is set from upstream ci, overwrite params
+echo "Upstream PARAMETERS set to $PARAMETERS"
+export params=(${PARAMETERS:-gcp 15 120})
+echo "params is $params"
+
+export CLOUD=${params[0]:-"gcp"}
+export REPLICAS=${params[1]:-"15"}
+export JOB_ITERATIONS=${params[2]:-"120"}
+
+echo "Testing with $CLOUD $REPLICAS $JOB_ITERATIONS"
+
+# Install dittybopper to check resource usage
+install_dittybopper
+
+if [ $? -eq 0 ]; 
+then 
+    # Cluster health check prior to testing
+    python ./utils/ocp_utils.py cluster_health_check
```

Comment:
Let's do something like [this line](https://github.com/openshift/svt/blob/7364b2b76b8ddc23fd74ca11b3c6c9f81ddeddcb/openshift_scalability/scripts/pod_density/pod_density.sh#L62) so we can take out  (if(sys.argv[1]=='cluster_health_check'):) in ocp_utils.py and just call the function directly